README.md

```diff
@@ -1 +1,70 @@
-# Math-Magicians
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```

Comment:
- [ ] Kindly update the readme file, currently you have default create-react-app README 😉